samples/subsys/ipc/ipc_service/README.rst

```diff
@@ -0,0 +1,83 @@
+.. _IPC_Service_sample:
+
```

Comment:
Any specific reason to create a new sample instead of updating the existing one?

Reply:
> It looks that the user can implement inter-processor communication using selected layer of communication. It should be possible to create a sample using shared memory and IPM directly. Another option is to use OpenAMP directly (what `openamp` sample presents). One more option is to use IPC Service created with this patch.
> 
> Using OpenAMP directly seems to be more flexible than using IPC Service, but using IPC Service is easier. Because of that I think both samples are valuable. We could simply remove `openamp` sample if we decide its value is lower than maintenance cost.
in my pov using IPC service is more flexible as it offer an API to share the RPmsg between drivers and applications.
then no strong opinion on the sample just slight preference for the update of the existing sample to help to illustrate genericity.